config.json

```diff
@@ -399,7 +399,15 @@
       "core": false,
       "difficulty": 5,
       "slug": "prime-factors",
-      "topics": null,
+      "topics": [
+        "arrays",
+        "conditionals",
+        "integers",
+        "lists",
+        "loops",
+        "mathematics",
+        "recursion"
```

Comment:
Just for your consideration: use of `recursion` is discouraged in Java

Reply:
Great suggestions! :) But I agree with @m-a-ge, I wouldn't add `recursion`. You sould be able to solve the problem without using recursion and using iteration is more suitable for Java :) What do you think? :)